Cargo.toml

```diff
@@ -25,6 +25,7 @@ serde = "1.0.197"
 serde_json = "1.0.114"
 chrono = "0.4.38"
 sea-orm = { version = "1.0.0-rc.5", features = [ "sqlx-postgres", "runtime-tokio-rustls", "macros" ] }
+sqlx = { version = "0.5", features = [ "postgres", "runtime-tokio-rustls" ] }
```

Comment:
btw, why are you adding sqlx as a dependency tho?

Reply:
I was trying to setup the db, and on cli, I don't know why but sqlx error was coming so I added it. 
I forgot to review. I will remove it.